src/main/resources/db/migration/V3__Create_klage_schema.sql

```diff
@@ -104,6 +131,37 @@ CREATE TABLE klage.vedtak
             REFERENCES klage.klagebehandling (id)
 );
 
+CREATE TYPE adressetype AS ENUM ('NORSK', 'UTENLANDSK');
+
+CREATE TABLE klage.adresse
+(
+    id                  UUID PRIMARY KEY,
+    adressetype         adressetype NOT NULL,
+    adresselinje1       TEXT,
+    adresselinje2       TEXT,
+    adresselinje3       TEXT,
+    postnummer          VARCHAR(4),
+    poststed            TEXT,
+    land                VARCHAR(2)
+);
+
+CREATE TABLE klage.brevmottaker
```

Comment:
Oi, denne ble komplisert.. En koblingstabell mellom tre andre tabeller? Et vedtak kan ha flere mottagere som kan ha flere adresser? Eller? Denne sleit jeg litt med. Kan den gjøres enklere?

Reply:
Adresse kan godt puttes inn i brevmottaker-tabellen. Så er spørsmålet om vi skal ha en egen id-tabell. Alternativet er å nullable-felter for fødselsnummer, organisasjonsnummer og virksomhetsnummer. Kommer bare an på hvor normalisert det skal være.